test/test_sanity.py

```diff
@@ -527,9 +527,12 @@ def test_emconfig(self):
   def test_emcc_ports(self):
     restore_and_set_up()
 
+    # validating that ports are valid
+    ports.validate_ports()
```

Comment:
I think you can remove this now that we do this on load.

Reply:
Ok. Sure.